docs/flow/modin/data_management/factories.rst

```diff
@@ -0,0 +1,19 @@
+Factories Module Description
+""""""""""""""""""""""""""""
+
+Brief description
+'''''''''''''''''
+Modin has several execution backends. Calling any DataFrame API function will end up in some backend-specific method. The responsibility of dispatching high-level API calls to backend-specific function lays on :doc:`QueryCompiler </flow/modin/backends/query_compiler.rst>`, which is determined by the factory of the corresponding backend. The mission of this module is to route IO function calls from the API level to its actual backend-specific implementations, which build `QueryCompiler` of the appropriate backend.
```

Comment:
```suggestion
Modin has several execution backends. Calling any DataFrame API function will end up in some backend-specific method. The responsibility of dispatching high-level API calls to backend-specific function lays on `QueryCompiler`, which is determined by the factory of the corresponding backend. The mission of this module is to route IO function calls from the API level to its actual backend-specific implementations, which build `QueryCompiler` of the appropriate backend.
```

Reply:
Let's not insert the link for now because it doesn't exist based on the tree in #2952.